patterns.md

```diff
@@ -204,14 +204,10 @@ by using `~>` instead of `>=` if at all possible.
 > [Isolate](https://github.com/jbarnette/isolate) which do a great job of
 > managing a complex version manifest for many gems.
 
-If you want to allow prereleases and regular releases use a compound
-requirement:
+If you want to only allow the use of prereleases, use a compound requirement:
 
     # gemspec
-    spec.add_runtime_dependency 'library', '>= 2.0.0.a', '< 3'
-
-Using `~>` with prerelease versions will restrict you to prerelease versions
-only.
+    spec.add_runtime_dependency 'library', '>= 2.0.0.a', '< 2.0.0'
```

Comment:
I'd leave the previous `spec.add_runtime_dependency 'library', '>= 2.0.0.a', '< 3'` in place, and state

> If you want to allow the use of prereleases, make sure to use a prerelease requirement (one that includes non numerical characters):

Reply:
Good point! Since this affects all operators (not just `~>`), I'll move it to its own paragraph.